app/selectors/autocomplete.js

```diff
@@ -76,6 +76,7 @@ export const filterMembersInChannel = createSelector(
                 return (p.delete_at === 0 && (
                     p.username.toLowerCase().includes(matchTerm) || p.email.toLowerCase().includes(matchTerm) ||
                     p.first_name.toLowerCase().includes(matchTerm) || p.last_name.toLowerCase().includes(matchTerm) ||
+                    `${p.first_name.toLowerCase()} ${p.last_name.toLowerCase()}`.includes(matchTerm) ||
```

Comment:
This can not be immediately clear it's the full name. I'd suggest you create a variable right before the return statement 
```
const fullName = `${p.first_name.toLowerCase()} ${p.last_name.toLowerCase()}`
```
and then go ahead to do the `fullName.includes(matchTerm)